src/editor/Objects/BoxController.tsx

```diff
@@ -0,0 +1,31 @@
+import { useRef, useState } from 'react';
+import * as THREE from 'three';
+import { ThreeElements, useFrame } from '@react-three/fiber';
+
+function BoxController(props: ThreeElements['mesh']) {
+  const ref = useRef<THREE.Mesh>(null!);
+  const [hovered, hover] = useState(false);
+  const [clicked, click] = useState(false);
+
+  useFrame((_, delta) => {
+    ref.current.rotation.z += delta;
+    ref.current.rotation.y += delta;
+    ref.current.rotation.x += delta;
+  });
```

Comment:
`useFrame` mysle zeby na razie usunac dla wszystkich ksztaltow. ja to dodalem bardziej zeby cos sie dzialo na scenie, ale docelowo nie chcemy zeby byla jakas animacja na start